Empire/Empire.cpp

```diff
@@ -1182,7 +1198,7 @@ void ProductionQueue::Update() {
         sim_pp_in_stockpile = CalculateNewStockpile(m_empire_id, sim_pp_in_stockpile,
                                                     available_pp, allocated_pp,
                                                     allocated_stockpile_pp);
-        sim_available_stockpile = std::min(sim_pp_in_stockpile, pp_use_limit->Current());
+        sim_available_stockpile = std::min(sim_pp_in_stockpile, stockpile_limit);
```

Comment:
It may have seemed like this min operation was still needed here simply because of the above-mentioned bug with ```CalculateNewStockpile```, but I don't think it is.

Reply:
I think it makes sense... What is available for use is the lesser of what's in the stockpile and the per-turn withdrawal limit.